lms/templates/main.html

```diff
@@ -107,13 +107,13 @@
 % if not disable_window_wrap:
   <div class="window-wrap" dir="${static.dir_rtl()}">
 % endif
-    <a class="nav-skip" href="<%block name="nav_skip">#content</%block>">${_("Skip to main content")}</a>
+    <a class="nav-skip" href="#content">${_("Skip to main content")}</a>
```

Comment:
It looks like Dave St.Germain was the original author of this line and he is no longer with us. My guess is that he was generating these `href`'s dynamically, which was causing the skip links to only work on some pages. My change makes this reference static so it will nearly always match up with the content area.